server/src/database/config/build.js

```diff
@@ -5,8 +5,12 @@ const connection = require('./connection');
 
 const dbBuild = () => {
   const dbFile = readFileSync(join(__dirname, 'build.sql')).toString();
-
-  return connection.query(dbFile);
+  const category = readFileSync(join(__dirname, 'category.sql')).toString();
+  const fakeData = readFileSync(join(__dirname, 'fakeData.sql')).toString();
+  return connection
+    .query(dbFile)
+    .then(connection.query(category))
+    .then(() => connection.query(fakeData));
```

Comment:
**It is a good way to build all files :+1:** 
**but there is a better syntax to achieve this directly  :** 
```
 let sql = readFileSync(join(__dirname, 'build.sql')).toString();

  sql += readFileSync(join(__dirname, 'category.sql')).toString();
  return connection.query(dbFile);	  if (process.env.NODE_ENV === 'test') {
    sql += readFileSync(join(__dirname, 'fakeData.sql')).toString();
  }
  return connection.query(sql);
```